### DISCUSSION TITLE
can i use this with dynamic dns?

### DISCUSSION THREAD

**Discussion by ricky-hammers** [GerryDazoo/Slinger#257] [General]
Amazing work here, just very confusing.  A tutorial would go a long way.  Especially for people without static IPs.  I configured the config.ini but can't get vlc to display a picture on my home network, let alone remotely.  Disappointed for now, but at least the ground work is here and someone will eventually be able to put it in clear step by step instructions for idiots like me.

**Comment by sp222222** [comment]
you can use ddns sure ! put that in the config.ini instead of the IP address for wherever the slingbox itself is

**Comment by sp222222** [comment]
you can use ddns sure ! put that in the config.ini instead of the IP address for wherever the slingbox itself is

**Comment by DoogieHowitzerWMD** [comment]
If you have your server in the same location as your slingboxes, you would use the DDNS to access your streams, so it would be: **http://yourdns:yourexternalport/yourURLbase**

If you have your server in a different location than your slingboxes, you specify the DDNS of the network where your slingboxes are located as the IP address in the config file, as well as the external port that slingbox is using. Then, to access your stream, you would do the same as the above, but use another DDNS and the external ports of the network where your server is located.
